ethereum.star

```diff
@@ -12,6 +12,11 @@ def run(plan, args):
                     "el_type": "geth",
                     "cl_type": "lighthouse",
                     "el_extra_params": ["--gcmode archive"],
+                    "cl_extra_params": [
+                        "--disable-optimistic-finalized-sync",
+                        "--disable-backfill-rate-limiting",
```

Comment:
It might be worth adding some context around these flags. I'm not sure what they are for.

Reply:
I'll add them with comments, but for more context, I got the idea to use the flags from this [discussion](https://github.com/ethereum/go-ethereum/issues/27482#issuecomment-1594305329)